packages/devtools_shared/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 4.0.0
```

Comment:
there was a recent commit to master for version 3.0.2. It hasn't been released yet, so can you merge and move the entry from 3.0.2 to 4.0.0? and then you can remove the 3.0.2 section

Reply:
This comment still stands